docs/modules/tiered-storage/pages/performance.adoc

```diff
@@ -0,0 +1,53 @@
+= Performance
+
+== Throughput vs Access Pattern
+
+Tiered Storage is sensitive to disk reads and therefore performs best when operating with entries retrieved mostly from the memory.
+The higher the probability of hitting an entry in the memory, the closer Tiered Storage gets to the throughput of HD.
+This means random access pattern is the worst for Tiered Storage, while predictable access patterns working mostly with the hot cache fitting in the memory is the best.
+
+== Impact of the Access Latency of the Devices
+
+Tiered Storage reads from the device synchronously, making the access latency of the device directly translating to throughput when the device needs to be touched.
+This rules out all high latency devices (HDDs, SAN drives, cloud block storages etc) from the list of the recommended types.
+
+== Compaction is the Driver of the Performance
+
+Compaction runs synchronously before/after the IMap operations making it the main driver/limiter of the performance of Tiered Storage.
+There are a few aspects that determine the impact of compaction on the performance of Tiered Storage:
+
+- *Frequency of the compaction*: The less frequently it runs, the less impact it has on the latency, therefore, on the throughput.
+The compactor gets active when the used space or the amount of garbage reaches the configured threshold.
+Then it runs incrementally and time-boxed to make the smallest impact possible on latency.
+
+- *Efficiency of the compaction*: The compaction is the most efficient when it can just clean up disk space without relocating alive entries inside the address space of Tiered Storage.
+This can be impacted by trading off disk space for compaction performance, see later.
+
+- *Hot cache thrashing*: The compactor Tiered Storage employs relocates the alive entries it encounters with to the memory where the hot cache resides.
+Since the memory consumption of the HybridLog is fixed, this means records from the memory hot cache will be accessible only from the disk until the next access.
+This means that the compactor replaces hot entries with cold entries temporarily.
+
+== Compaction vs Disk Space Trade-Off
```

Comment:
Do we mention any parameters to manage the trade-off?

Reply:
What do you mean? The configuration page describes all parameters for the disk, including its capacity.

Reply:
Well, it means that if you configure the capacity larger, the compaction becomes more efficient. There is a section about this in the performance page, I wouldn't repeat it here.